File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p03434/p03434_num3_parsed.py
# Taking input for the number of cards
N = int(input())

# Taking input for the values on the cards
cards = list(map(int, input().split()))

# Sorting the cards in descending order
cards.sort(reverse=True)

# Initializing variables for Alice's and Bob's scores
alice_score = 0
bob_score = 0

# Distributing the cards between Alice and Bob
for i in range(N):
    if i % 2 == 0:
        alice_score += cards[i]
    else:
        bob_score += cards[i]

# Calculating the difference between Alice's and Bob's scores
score_difference = alice_score - bob_score

# Printing the result
print(score_difference)

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p03434/p03434_num3_parsed.cpp
#include <iostream>
#include <vector>
#include <algorithm>

int main() {
    int N;
    std::cin >> N;

    std::vector<int> a(N);
    for (int i = 0; i < N; i++) {
        std::cin >> a[i];
    }

    std::sort(a.begin(), a.end(), std::greater<int>());

    int aliceScore = 0;
    int bobScore = 0;
    for (int i = 0; i < N; i++) {
        if (i % 2 == 0) {
            aliceScore += a[i];
        } else {
            bobScore += a[i];
        }
    }

    std::cout << aliceScore - bobScore << std::endl;

    return 0;
}